Ошибка в модуле @salute/types

Error: Cannot find module '.../node_modules/@salutejs/types/dist/index.d.js'. Please verify that the package.json has a valid "main" entry

Сейчас:
"main": "dist/index.d.js",
дб
"main": "dist/index.js",

--- a/packages/types/package.json
+++ b/packages/types/package.json
@@ -4,7 +4,7 @@
   "description": "Scenario types for Salute family",
   "author": "SberDevices Frontend Team <sberdevices.frontend@gmail.com>",
   "license": "Sber Public License at-nc-sa v.2",
-  "main": "dist/index.d.js",
+  "main": "dist/index.js",
   "typings": "dist/index.d.ts",
   "homepage": "https://github.com/sberdevices/salutejs#readme",
   "repository": {

